src/scripts/components/VariableMessage.jsx

```diff
@@ -424,7 +663,9 @@ VariableMessage.propTypes = {
   previewLabel: PropTypes.string,
   name: PropTypes.string.isRequired,
   reset: PropTypes.bool,
-  variables: PropTypes.array.isRequired,
+  data: PropTypes.array,
```

Comment:
What's the preference for renaming this from `variables` --> `data`. The new name seems unclear when looking at this component compared to the previous name.

If this prop is required based on the standard function of the component let's mark it that way.